catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrMetacardClientImpl.java

```diff
@@ -623,8 +618,8 @@ public MetacardImpl createMetacard(SolrDocument doc) throws MetacardCreationExce
   @Override
   public List<SolrInputDocument> add(List<Metacard> metacards, boolean forceAutoCommit)
       throws IOException, SolrServerException, MetacardCreationException {
-    if (metacards == null || metacards.size() == 0) {
-      return null;
+    if (metacards == null || metacards.isEmpty()) {
```

Comment:
nit: could use CollectionUtils here for a cleaner read.  Same below on identifiers